src/components/stack-chart/index.js

```diff
@@ -181,7 +193,9 @@ class StackChartGraph extends React.PureComponent<Props> {
                   interval,
                   thread,
                   pages,
-                  stackTimingByDepth,
+                  threadIndex,
```

Comment:
I think this can be reverted too along with the props.

Reply:
I think we need threadIndex and getMarker in `_getHoveredStackInfo`